src/MetaCompilation.Analyzers/Core/xlf/MetaCompilationResources.cs.xlf

```diff
@@ -5,15 +5,15 @@
       <trans-unit id="AnalyzerDescription">
         <source>Type names should be all uppercase.</source>
         <target state="new">Type names should be all uppercase.</target>
-        <note>An optional longer localizable description of the diagnostic.</note>An optional longer localizable description of the diagnostic.</trans-unit>
+        <note />An optional longer localizable description of the diagnostic.</trans-unit>
```

Comment:
That might have been it. I deleted xlf files in source, but probably didn't clean obj dir.